packages/interface-internal/src/transport-manager.ts

```diff
@@ -2,13 +2,41 @@ import type { AbortOptions, Connection, Listener, Transport, TransportManagerDia
 import type { Multiaddr } from '@multiformats/multiaddr'
 import type { ProgressOptions } from 'progress-events'
 
+/**
+ * @packageDocumentation
+ *
+ * The `TransportManager` module handles the management of transport protocols in a libp2p network.
+ * It is responsible for adding, removing, and managing transport connections used for peer communication.
```

Comment:
It's responsible for managing the transports themselves - dialling, querying addresses and listening.